packages/flutter/lib/src/widgets/tap_and_drag_gestures.dart

```diff
@@ -722,15 +722,6 @@ class TapAndDragGestureRecognizer extends OneSequenceGestureRecognizer with _Tap
   ///  * [DragGestureRecognizer.dragStartBehavior], which includes more details and an example.
   DragStartBehavior dragStartBehavior;
 
-  /// The frequency at which the [onDragUpdate] callback is called.
-  ///
-  /// The value defaults to null, meaning there is no delay for [onDragUpdate] callback.
-  ///
-  /// See also:
-  ///   * [TextSelectionGestureDetector], which uses this parameter to avoid excessive updates
-  ///     text layouts in text fields.
-  Duration? dragUpdateThrottleFrequency;
```

Comment:
This is public API, which makes removing it a breaking change because some apps might be using it.

Reply:
Yes, this will need to be deprecated instead. You can start to ignore the value, but it needs to stay in the API for a while.

Check out other examples of deprecation in the code (search for "@Deprecated"), and look at https://github.com/flutter/flutter/wiki/Tree-hygiene#deprecations

Be sure to update the documentation comment to make it obvious that the field is deprecated, and that it will be ignored.

Reply:
I wasn't aware `TapAndDragGestureRecognizer` could be used by other apps. If that's the case then perhaps we keep `dragUpdateThrottleFrequency` and just make the text_selection.dart changes?

Reply:
@loune That sounds good to me. I think you can also remove the comments in tap_and_drag_gestures.dart related to this issue.

```
/// See also:
///   * [TextSelectionGestureDetector], which uses this parameter to avoid excessive updates
///     text layouts in text fields.
```